### DISCUSSION TITLE
Is there a way to add extensions to chromebook?

### DISCUSSION THREAD

**Discussion by RapidOnGfuel** [3kh0/ext-remover#73] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm curious about this as this would be a huge help relative to school. Is it possible to like "force" add an extension?

**Comment by 3kh0** [comment, marked as answer]
No

**Reply by chickennu99et** [reply]
well actually yes
you can by downloading the zip file of the extension and loading unpacked but you need dev mode

**Reply by Aka-but-nice** [reply]
all you had to say was no

**Reply by chickennu99et** [reply]
ye
